### ISSUE TITLE
No target with given id found

### ISSUE THREAD

**Issue by justingolden21** [GoogleChrome/lighthouse#15161]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

Locally. Cannot upload as it is sensitive info.

### What happened?

![image](https://github.com/GoogleChrome/lighthouse/assets/30274440/1fd674e7-b8c7-4f7f-ad5f-e7024accffd4)

Ran lighthouse and it kept failing. Updated chrome and it kept succeeding. Restored stashed changes of `data-no-lazy=""` (which I've been repeatedly stashing and restoring and has never crashed) and this one time after restoring, refreshing the page, and running lighthouse, it crashed like this.

Clicking "analyze page load" again results in a successful run.

### What did you expect?

Lighthouse report to run

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

Not in report

### Chrome Version

Version 114.0.5735.110 (Official Build) (64-bit)

### Node Version

N/A

### OS

Windows 10

### Relevant log output

```shell
Protocol error (Target.attachToTarget): No target with given id found


Channel: DevTools
Initial URL: https://mywebsite.com
Chrome Version: 114.0.0.0
Stack Trace: f: Protocol error (Target.attachToTarget): No target with given id found
    at devtools://devtools/bundled/devtools-frontend/front_end/services/puppeteer/puppeteer.js:1:4154
    at new Promise (<anonymous>)
    at Pd.send (devtools://devtools/bundled/devtools-frontend/front_end/services/puppeteer/puppeteer.js:1:4087)
    at Pd._createSession (devtools://devtools/bundled/devtools-frontend/front_end/services/puppeteer/puppeteer.js:1:5363)
    at devtools://devtools/bundled/devtools-frontend/front_end/services/puppeteer/puppeteer.js:1:188840
    at Array.map (<anonymous>)
    at PuppeteerConnectionHelper.connectPuppeteerToConnection (devtools://devtools/bundled/devtools-frontend/front_end/services/puppeteer/puppeteer.js:1:188830)
    at i (devtools://devtools/bundled/devtools-frontend/front_end/entrypoints/lighthouse_worker/lighthouse_worker.js?remoteBase=https%3A%2F%2Fchrome-devtools-frontend.appspot.com%2Fserve_file%2F%401c828682b85bbc70230a48f5e345489ec447373e%2F:1:2087)
    at async self.onmessage (devtools://devtools/bundled/devtools-frontend/front_end/entrypoints/lighthouse_worker/lighthouse_worker.js?remoteBase=https%3A%2F%2Fchrome-devtools-frontend.appspot.com%2Fserve_file%2F%401c828682b85bbc70230a48f5e345489ec447373e%2F:1:2967)
```

**Comment by justingolden21** [comment]
This happens to many sites and production ones too. I can reproduce this many times in a row on chrome incognito. I don't have canary installed at the moment.

It once again stopped working since I created this ticket.

It worked and then weeks later it didn't and then weeks later it did and then it didn't and Q was like try updating so I updated and it worked, but now today it doesn't work again and chrome is up to date. Very fickle.

**Comment by adamraine** [comment]
I was able to reproduce this on https://espn.com although it is flaky like you said. This looks similar to #15026 where Puppeteer is having trouble connecting to certain targets. I haven't reproduced this on the latest Chrome canary yet so it's possible that @paulirish's fix also addressed this.

**Comment by lemanschik** [comment]
Only wanted to provide feedback as i was getting this bug even in Chromium Builds and not only Google Chrome builds. I reviewed Pauls assumptions and it LGTM+1 for me. I Will post here if i should encounter so this issue 

closes: #15206

**Comment by adamraine** [comment]
I can't reproduce this in the latest Chrome, closing unless this pops up again in the near future
